audioplayer.cpp

```diff
@@ -0,0 +1,92 @@
+#include "audioplayer.h"
+
+vector<string> AudioPlayer::GetAvailableTracks(const string& folder) {
+  vector<string> names;
+  DIR* directory = opendir(folder.c_str());
+
+  if (directory == nullptr) {
+	throw invalid_argument("Unable to find directory");
+  }
+
+  dirent* ent;
+  while ((ent = readdir(directory)) != nullptr) {
+	if ((strcmp(ent->d_name, "..")) && (strcmp(ent->d_name, "."))) {
+	  names.emplace_back(ent->d_name);
```

Comment:
There are troubles with indentation - somewhere it's 4, somewhere it's 2. Try to fully format the file and later follow the identation. You need to setup your tab identation just like in your code style in settings

Reply:
yep, fixed